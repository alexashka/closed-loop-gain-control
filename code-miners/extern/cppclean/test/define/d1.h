
void SomePublicFunction(void);

void OnlyDeclared(void);

class SomeClass {
  void Method();
};
