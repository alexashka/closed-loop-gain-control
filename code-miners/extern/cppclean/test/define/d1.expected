Processing d1.cc
d1.cc:11: SomeFunction1 not found in expected header d1.h or any other directly #included header
d1.cc:15: expected to find SomeOtherFunction1 in d1.h, but found in other_defined1.h
d1.h:4: OnlyDeclared declared but not defined
